CMakeLists.txt

```diff
@@ -2792,6 +2792,8 @@ if(QML)
     # The following sources need to be in this target to get QML_ELEMENT properly interpreted
     src/control/controlmodel.cpp
     src/control/controlsortfiltermodel.cpp
+    # needed for qml/qmlautoreload.cpp
```

Comment:
We normally don't have such comments, remove?

Reply:
FYI I'm using autoreloader in #13082

Reply:
Thanks for the headsup. well then it needs to be linked in both anyways.